.github/workflows/audit-security.yml

```
@@ -0,0 +1,48 @@
+# Copyright 2021 The Engula Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Audit Security
+
+on:
+  push:
+    paths:
+      - '**/**/Cargo.toml'
```

```suggestion
      - '**/Cargo.lock'
```

If we audit lock file, watch exactly on lock files. Also if we generate lock file at root only, you may replace with `Cargo.lock`.

---

OK I get your idea that check on toml and generate lock file below. It's another approach and more self-contained.

Either way `**/**/` can be replaced in `**/`

---

ok, you are right, lock file is enough